core/Loc.h

```diff
@@ -29,7 +29,7 @@ CheckSize(LocOffsets, 6, 1);
 class Loc final {
     struct {
         LocOffsets offsets;
-        unsigned int fileRef : 16;
+        unsigned int fileRef : 24;
```

Comment:
This is not a u4. is that ok?

Reply:
It's enough for now, I'll followup with doing this properly soon